### ISSUE TITLE
show download speed in title like qbitorrent

### ISSUE THREAD

**Issue by zdmgmail** [qutebrowser/qutebrowser#5955]
![qbar](https://user-images.githubusercontent.com/50515839/101705729-8be46680-3ac2-11eb-8d51-9446cb9626bc.png)

qbitorrent can change title and show download/upload speed,  see it in statusbar really helpful,
could you implement this in qutebrowser?

(And how to not show download speed in qutebrowser?)

**Comment by The-Compiler** [comment]
I don't really see the point of this to be honest - can you elaborate? It's not like qutebrowser is a download manager or its primary purpose is downloading files.

**Comment by user202729** [comment]
For the second question, usually you would be able to configure it by setting a templated string in `:set something.format` or `:set something_format`, but the one for download item in particular is hard coded in `qutebrowser/browser/downloads.py` (`'{index}: {name} [{speed:>10}|{down}]{errmsg}'`).

**Comment by BurnyBoi** [comment]
The point is that if you don't have the window open, you can still see the download progress at a glance. While no browser's purpose is a download manager, we've all gone to download a large file before, realized it's going to take a while, and left the browser open for it to finish. Being able to keep track of it no matter if the browser is visible does have utility.

**Comment by user202729** [comment]
Yes, Chrome updates its icon on the taskbar (at least on Windows) to show the download progress too.
I don't know if it's possible with Qt, or on other operating systems, or what should be done if there are multiple ongoing downloads.
There's already `window.title_format`, but there's no download-related items to show there.
